browser/ui/commands/accelerator_service.cc

```diff
@@ -210,18 +210,20 @@ void AcceleratorService::ResetAcceleratorsForCommand(int command_id) {
 }
 
 void AcceleratorService::ResetAccelerators() {
-  accelerators_ = default_accelerators_;
-
   std::vector<int> commands;
   for (const auto command : GetCommands()) {
     pref_manager_.ClearAccelerators(command);
     commands.push_back(command);
 
     // Make sure we add all the accelerators back.
-    for (const auto& accelerator : accelerators_[command]) {
+    for (const auto& accelerator : default_accelerators_[command]) {
       pref_manager_.AddAccelerator(command, accelerator);
     }
   }
+
+  // Load the default accelerators back.
+  accelerators_ = pref_manager_.GetAccelerators();
```

Comment:
Sorry, can you explain what makes crash and how it's fixed?
For me, before & after seems same.

Reply:
So the `default_accelerators` contains some Chromium shortcuts we don't support. We reset all the commands we support (the result of `GetCommands`).

The problem was, that when we just assign `accelerators = default_accelerators` the unsupported Chromium one was included - and that meant we'd crash when trying to display the name of the unsupported command.

Instead, we set our accelerators to what's stored in the pref after a reset :smile:

Reply:
ah, got it!